pride_cv.obo

```diff
@@ -4424,6 +4424,14 @@ def: "Data-independent acquisition parallel accumulation-serial fragmentation, P
 is_a: PRIDE:0000450 ! Data-independent acquisition
 created_by: yperez
 
+[Term]
+id: PRIDE:0000651
+name: Absolute quantification
+def: "Absolute quantification methods involve accurately determining the exact concentration or amount of a specific analyte in a sample through the use of known standards or reference materials, enabling precise measurement of absolute quantities" [PRIDE:PRIDE]
```

Comment:
What do you think about this: `Absolute quantification precisely measures the concentration of a specific analyte in a sample` The idea is to remove the known standard idea because you may try to compute absolute quant without known standards.

Reply:
ok..we can add it instead.